tests/candev/main.c

```diff
@@ -219,12 +219,11 @@ if (IS_ACTIVE(CONFIG_USE_LOOPBACK_MODE)) {
     /* set to loopback test mode */
     canopt_state_t mode = CANOPT_STATE_LOOPBACK;
     candev->driver->set(candev, CANOPT_STATE, &mode, sizeof(mode));
-
-    /* do not care, receive all message id */
+}
+    /* Add filters here when needed. Currently do not care, receive all message id */
```

Comment:
Maybe add an example on how to use it? The reason for writing this candev application was because the CAN system was extremely complex before. This showed a very basic and easy way to use CAN in RIOT. Now that you've enabled another feature, maybe this feature can also be demonstrated in the candev test?